src/main/java/it/gov/pagopa/payhub/activities/connector/ionotification/mapper/NotificationRequestMapper.java

```diff
@@ -7,17 +7,20 @@
 import org.springframework.context.annotation.Lazy;
 import org.springframework.stereotype.Service;
 
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.stream.Collectors;
 
+import static it.gov.pagopa.payhub.activities.util.Utilities.centsAmountToEuroString;
 import static java.util.stream.Collectors.groupingBy;
 
 @Service
 @Lazy
 public class NotificationRequestMapper {
 
-    public List<NotificationRequestDTO> map(List<PaymentOptionDTO> paymentOptions, Long orgId, Long debtPositionTypeOrgId, IONotificationDTO ioNotificationDTO, NotificationRequestDTO.OperationTypeEnum operationType) {
+    public List<NotificationRequestDTO> map(List<PaymentOptionDTO> paymentOptions, Long orgId, Long debtPositionTypeOrgId, String dpDescription, IONotificationDTO ioNotificationDTO) {
```

Comment:
let's ask the entire DebtPosition instead of just the dpDescription!
This way we could add simply any new placeholder if needed